DASH-MPD-PATCH-BASIC.xsd

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:patch="urn:ietf:params:xml:schema:patch-ops" xmlns="urn:mpeg:dash:schema:mpd-patch:2020" targetNamespace="urn:mpeg:dash:schema:mpd-patch:2020" elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <!-- Import of patch operations from RFC5261 -->
+    <xs:import namespace="urn:ietf:params:xml:schema:patch-ops" schemaLocation="https://www.iana.org/assignments/xml-registry/schema/patch-ops.xsd"/>
+    <xs:element name="Patch" type="PatchType"/>
+    <!-- Patch -->
+    <xs:complexType name="PatchType">
+        <xs:choice minOccurs="0" maxOccurs="unbounded">
```

Comment:
`xs:choice` used here in place of typical `xs:sequence` because the ordering of the `add`/`remove`/`replace` operations can technically be significant allowing variations across implementations. Open to better methods of signalling this.